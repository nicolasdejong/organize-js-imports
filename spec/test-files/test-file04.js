import 'zoo';
import 'someverylongname';
import x,y from 'xyz';
import a from 'abc';
