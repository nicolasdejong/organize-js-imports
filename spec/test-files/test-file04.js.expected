import 'someverylongname';
import 'zoo';

import a    from 'abc';
import x, y from 'xyz';

